frontend/src/views/MockExams/MockExamsQuestion.tsx

```diff
@@ -0,0 +1,309 @@
+        </Typography>
+      </Flex>
+      <Flex flexDirection="column" flex="1">
+        {state.showMedia && (
+          <MediaContainer
+            hasVideo={hasVideo}
+            mediaURL={question.mediaReference}
+            onVideoEnd={startAnsweringQuestion}
+          />
+        )}
+        <Typography
+          variant="h6"
+          align="center"
+          color={theme.palette.text.primary}
+          sx={{
+            width: '85%',
+            margin: 'auto',
+            marginTop: state.showMedia ? '8px' : 'auto',
+            marginBottom: state.showMedia ? '8px' : 'auto',
+            paddingTop: '8px',
+            paddingBottom: '8px',
+          }}
+        >
+          {question.question}
+        </Typography>
+      </Flex>
+      <Flex
+        alignItems="center"
+        style={{ margin: '16px 0 32px 0' }}
+        flexDirection="column"
+      >
+        <Flex
+          style={{ margin: '8px auto 8px auto', padding: 'auto' }}
+          flexDirection={answers.length > 2 ? 'column' : 'row'}
+        >
+          {answers.map((answer) => {
+            return (
+              <FormControlLabel
+                key={answer.id}
+                value={answer.id}
+                control={
+                  <Checkbox
+                    checked={selectedAnswer === answer.id}
+                    color="primary"
+                    onChange={handleAnswerSelection}
+                    value={answer.id.toString()}
+                    inputProps={{ type: 'radio' }}
+                    name="answer"
+                  />
+                }
+                label={answer.answerContent}
+                labelPlacement="end"
+              />
+            );
+          })}
+        </Flex>
+        <LinearProgress
+          variant="determinate"
+          style={{
+            height: '16px',
+            borderRadius: '0.4rem',
+            width: '80%',
+            margin: '16px auto 16px auto',
+          }}
+          value={progress}
+        />
+        <Flex style={{ margin: '8px 0 8px 0' }}>
+          {state.showStartButton && (
+            <Button
+              variant="outlined"
+              style={{ margin: '8px 8px 0 auto' }}
+              onClick={handleReadyToAnswer}
+            >
+              {startLabel}
+            </Button>
+          )}
+          <Button
+            variant="contained"
+            style={{ margin: '8px auto 0 auto' }}
+            onClick={submitQuestion}
+            disabled={selectedAnswer === undefined}
+          >
+            {nextQuestionLabel}
+          </Button>
+        </Flex>
+      </Flex>
+    </Flex>
+  );
+};
+
+const MediaContainer = ({ mediaURL, hasVideo, onVideoEnd }: MediaInterface) => {
+  return hasVideo ? (
+    <StyledMediaWrapper elevation={1}>
+      {mediaURL && (
+        <StyledVideo
+          width={700}
+          src="/1A601.mov"
+          onEnded={onVideoEnd}
+          autoPlay
```

Comment:
Let's also add `muted`. Some browsers could prevent this from being played otherwise